sklearn/feature_selection/tests/test_feature_select.py

```diff
@@ -18,9 +19,10 @@
 
 from sklearn.datasets import make_classification, make_regression
 from sklearn.feature_selection import (
-    chi2, f_classif, f_oneway, f_regression, mutual_info_classif,
-    mutual_info_regression, SelectPercentile, SelectKBest, SelectFpr,
-    SelectFdr, SelectFwe, GenericUnivariateSelect)
+    chi2, f_classif, f_oneway, f_regression, abs_r_regression, r_regression,
```

Comment:
since we are modifying these imports, could make an import by line in a `black` style:

```python
from sklearn.feature_selection import (
    chi2,
    f_classif,
...
)